src/figmaTypes.ts

```diff
@@ -66,8 +139,8 @@ export interface Canvas extends Global {
 export interface FrameBase extends Global {
   /** An array of nodes that are direct children of this node */
   readonly children: ReadonlyArray<Node>;
-  /** Background color of the node */
-  readonly backgroundColor: Color;
```

Comment:
`backgroundColor` is still available in their response.

Reply:
Can you get it to come back in a request? I found no way to cause this to be sent back, and the docs mention this is deprecated:

```
Background color of the node. This is deprecated, as frames now support more than a solid color as a background. Please use the background field instead.
```